backend/app.py

```
@@ -1,55 +1,153 @@
-from flask import Flask, session
+from flask import Flask, session, jsonify
```

Ruff (0.11.9)

`flask.session` imported but unused

Remove unused import

(F401)